x-pack/plugins/security_solution/public/hosts/components/host_risk_score_table/translations.ts

```diff
@@ -29,8 +29,8 @@ export const HOST_RISK_TOOLTIP = i18n.translate(
   }
 );
 
-export const HOSTS_BY_RISK = i18n.translate('xpack.securitySolution.hostsRiskTable.hostsTitle', {
-  defaultMessage: 'Hosts by risk',
+export const HOST_RISK_TITLE = i18n.translate('xpack.securitySolution.hostsRiskTable.hostsTitle', {
+  defaultMessage: 'Host risk',
```

Comment:
Should we remove the translations of `xpack.securitySolution.hostsRiskTable.hostsTitle` in these files to make sure their translations are updated as well?
x-pack/plugins/translations/translations/ja-JP.json
x-pack/plugins/translations/translations/zh-CN.json
x-pack/plugins/translations/translations/fr-FR.json